CHANGELOG.adoc

```diff
@@ -40,7 +40,7 @@
 - https://github.com/eclipse-sirius/sirius-web/issues/3887[#3887] [diagram] Memoize diagram representation (improve performance when selecting an element on large diagram)
 - https://github.com/eclipse-sirius/sirius-web/issues/3987[#3987] [sirius-web] Transform `widgetFields` fragment to retrieve custom widget fields
 - https://github.com/eclipse-sirius/sirius-web/issues/3875[#3875] [sirius-web] Move explorer related code from `sirius-components-trees` to `sirius-web-application`
-
+- https://github.com/eclipse-sirius/sirius-web/issues/3972[#3972] [diagram] Added option to force display of separator bar header.
```

Comment:
The summary could be improved given the new design (`HeaderSeparatorDisplayMode.{ALWAYS,NEVER,IF_CHILDREN}`.

Reply:
Done